index.bs

```diff
@@ -2331,8 +2335,9 @@ browsingContext.InfoList = [*browsingContext.Info]
 browsingContext.Info = {
   children: browsingContext.InfoList / null,
   context: browsingContext.BrowsingContext,
+  opener: browsingContext.BrowsingContext / null,
```

Comment:
Could we call this `originalOpener` perhaps, to make it clearer it's not the same as `window.opener`?

Reply:
Updated